src/components/ParametersTable.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="deployment-parameters-table">
     <div class="deployment-parameters-table__search">
-      <ResultsCount :count="filtered.length" label="parameters" />
+      <ResultsCount :count="filtered.length" :label="toPluralString('parameter', filtered.length)" />
```

Comment:
ResultsCount already pluralizes the string. Pass the singular label and it will run it through toPluralString I believe